server/src/middlewares/index.js

```diff
@@ -0,0 +1,9 @@
+const middlewares = {
+  Query: {
+    //
+  },
```

Comment:
Are we not using the middlewares anymore? If that's the case, is there any reason to keep them around?  We could delete, and we have them in git history if needed later.

Reply:
Thanks, fixed.